src/roulette.py

```diff
@@ -934,3 +940,30 @@ def cycle(self, player: Player) -> None:
                     player.win(bet)
                 else:
                     player.lose(bet)
+
+
+class Simulator:
+    def __init__(self, game: Game, player: Player) -> None:
+        self.game = game
+        self.player = player
+        self.initDuration = 250
+        self.initStake = 100
+        self.samples = 50
+        self.durations: list[int] = []
+        self.maxima: list[int] = []
+
+    def session(self) -> list[int]:
+        self.player.stake = self.initStake
+        self.player.roundsToGo = self.initDuration
+        stake_values = []
+        while self.player.roundsToGo > 0 and self.player.playing():
```

Comment:
better initialize `roundsToGo` with `initDuration` and perform check from `playing`